### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by darekmer** [templaza/astroid-framework#604] [General]
Hello, do you also have issues with translations of options in the 'Layout Builder' within Astroid settings?

**Comment by Banuk1616** [comment]
Hello,
what problems do you have? I use Astroid 3.0.11 and the German language file, but I haven't noticed any problems. What are their problems?

**Comment by darekmer** [comment]
Some know that when Astroid was still managed by Joomdev, I translated Astroid into Polish. There was never a problem. There isn't one now either, but I don't know where the English translation in 'Layout' came from. I cleared the cache, even reinstalled Astroid. I have version 3.0.11.
e.g. polish translation:
```
TPL_ASTROID_ADD_SECTION = "Dodaj sekcję"
TPL_ASTROID_NEW_SECTION = "Nowa sekcja"
TPL_ASTROID_DUPLICATE_SECTION = "Powiel sekcję"
TPL_ASTROID_REMOVE_SECTION = "Usuń sekcję"
TPL_ASTROID_EDIT_SECTION = "Edytuj sekcję"
TPL_ASTROID_EDIT_SECTION_TITLE = "Kliknij i edytuj tytuł"
TPL_ASTROID_NEW_ROW = "Nowy wiersz"
TPL_ASTROID_REMOVE_ROW = "Usuń wiersz"
TPL_ASTROID_DUPLICATE_ROW = "Powiel wiersz"
TPL_ASTROID_EDIT_ROW = "Edytuj wiersz"
TPL_ASTROID_EDIT_GRID_ROW = "Edytuj siatkę"
TPL_ASTROID_SELECT_AN_ELEMENT = "WYBIERZ ELEMENT"
TPL_ASTROID_SELECT_GRID_LAYOUT = "WYBIERZ UKŁAD SIATKI"
TPL_ASTROID_EDIT_ELEMENT = "Edytuj element"
TPL_ASTROID_DUPLICATE_ELEMENT = "Powiel element"
TPL_ASTROID_EDIT_COLUMN = "Edytuj kolumnę"
TPL_ASTROID_MOVE_COLUMN = "Przenieś kolumnę"
TPL_ASTROID_ADD_ELEMENT_COLUMN = "Dodaj element"
TPL_ASTROID_MOVE_ROW = "Przenieś wiersz"
TPL_ASTROID_MOVE_ELEMENT = "Przenieś element"
TPL_ASTROID_REMOVE_ELEMENT = "Usuń element"
```
![layout](https://github.com/templaza/astroid-framework/assets/44267306/602d5dd1-984b-4329-b373-235aa105f626)

**Reply by sonvnn** [reply]
Thank you! I will soon update Translate code for Layout Builder. I will inform you as soon as It is ready.
